### PR TITLE
6 - Add entities

### PR DESCRIPTION
**Pull request by TibrExx**
- Add data starter
- Add postgres#6: Persistence
- Add entities
- Add data starter
- Add postgres#6: Persistence
- Add entities
- Add data starter
- Add postgres

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,5 @@
 * Slack: #spaceboys
 
 ## Start app
+* create `postgres via docker docker run --name postgres -e POSTGRES_PASSWORD=pass -e POSTGRES_USER=postgres -e POSTGRES_DB=volunteers -p 5432:5432 postgres`
```
Comment (frank-mueller-ps) [inline README.md:9]: Shouldnt we use a gradle plugin to run an embedded postgres instead of adding the dependency to docker?

Comment (izi01) [reply README.md:9]: this is for local testing only, right?

in production we'll use aws postgres

Comment (frank-mueller-ps) [reply README.md:9]: I thought using a gradle plugin for local testing like https://plugins.gradle.org/plugin/com.github.honourednihilist.gradle-postgresql-embedded could help. Not sure if all of us have docker installed.

Comment (Tim1389) [reply README.md:9]: For combined testing frontend and backend we will need something like a docker-compose. Therefore everyone should have docker in my opinion

**Pull request merged**